share/mk/bsd.compat.mk

```diff
@@ -195,10 +195,15 @@ LIBSOFTWMAKEENV= CPUTYPE=soft
 LIBSOFTWMAKEFLAGS=        -DCOMPAT_SOFTFP
 .endif
 
+.if defined(NEED_COMPAT) && ${NEED_COMPAT:MCHERI} && ${MACHINE_ABI:Mpurecap}
+.info "NEED_COMPAT=CHERI with default ABI == purecap, ignoring for ${.CURDIR}"
```

Comment:
This seems like it's going to be noisy?

Reply:
Yes, I will disable it once stuff works.